app/lib/atom_feed.dart

```diff
@@ -23,25 +23,25 @@ class FeedEntry {
   final String alternateTitle;
 
   FeedEntry(this.id, this.title, this.updated, this.authors, this.content,
-            this.alternateUrl, this.alternateTitle);
+      this.alternateUrl, this.alternateTitle);
```

Comment:
Oh, sorry. Got probably irritated by this github diff showing tool. We use normally rietveld which is much nicer for reviewing.